Makefile

```diff
@@ -210,53 +211,67 @@ test.integration: test.integration.dbless test.integration.postgres
 .PHONY: test.integration.enterprise
 test.integration.enterprise: test.integration.enterprise.postgres
 
-.PHONY: test.unit
-test.unit:
-	@go test -v -race \
+.PHONY: _test.unit
+_test.unit:
+	go test -v -race $(GOFLAGS) \
 		-covermode=atomic \
 		-coverpkg=$(PKG_LIST) \
 		-coverprofile=coverage.unit.out \
 		./internal/... \
-		./pkg/...
+		./pkg/... 2>/dev/null
 
-.PHONY: test.integration.dbless
-test.integration.dbless:
+.PHONY: test.unit
+test.unit:
+	@$(MAKE) _test.unit GOFLAGS="$(GOFLAGS)"
+
+.PHONY: test.unit.pretty
+test.unit.pretty:
+	@$(MAKE) _test.unit GOFLAGS="-json" 2>/dev/null | $(GOTESTFMT_CMD)
```

Comment:
`2 >/dev/null` is present in `_test.unit`, so `2 >/dev/null` would be redundant here.

Reply:
Changed